packages/cashier/src/types/shared/account.types.ts

```diff
@@ -1,23 +1,45 @@
 /* -------------------------------------------------------------------------- */
 /* ACCOUNT TYPES                                                              */
 /* -------------------------------------------------------------------------- */
-import { DetailsOfEachMT5Loginid } from '@deriv/api-types';
+import { DetailsOfEachMT5Loginid, TransferBetweenAccountsResponse } from '@deriv/api-types';
 
 export type TAccount = {
     balance?: string | number;
     currency?: string;
     disabled?: boolean;
+    error?: JSX.Element | string;
+    is_crypto?: boolean;
     is_dxtrade?: boolean;
     is_mt?: boolean;
     market_type?: string;
-    nativepicker_text: string;
+    nativepicker_text?: string;
     platform_icon?: string;
-    text: JSX.Element | string;
-    value?: string;
+    text?: JSX.Element | string;
+    value?: string | undefined;
 };
 
+export type TTransferAccount = {
+    account_type?: 'trading' | 'mt5' | 'wallet' | 'dxtrade' | 'binary';
+    sub_account_type?: string;
+    balance?: string;
+    currency?: string;
+    demo_account?: 0 | 1;
+    loginid?: string;
+    market_type?: 'financial' | 'synthetic';
+    mt5_group?: string;
+    landing_company_short?: 'bvi' | 'labuan' | 'malta' | 'maltainvest' | 'samoa' | 'svg' | 'vanuatu';
+};
+
+export type TTransferBetweensAccounts = TransferBetweenAccountsResponse;
```

Comment:
Export types from `Types` folder to prevent the import of the type from `@deriv/api-types` for better readability and easier to maintain.